### PR TITLE
#12 include versioning

### PR DESCRIPTION
**Pull request by jetzlstorfer**

### CODE REVIEW

File: site/tutorials/keptn-argo-cd-deployment/codelab.json [hunk 1]
```diff
@@ -19,8 +19,8 @@
     "quality-gates"
   ],
   "tags": [
-    "tutorial"
+    "keptn06x"
   ],
-  "feedback": "https://keptn.sh",
+  "feedback": "https://github.com/keptn/tutorials/tree/master/site/tutorials",
```
Comment (johannes-b) [inline site/tutorials/keptn-argo-cd-deployment/codelab.json:24]: Is this link change intended?

File: site/tutorials/keptn-notification-service_gen.md [hunk 2]
```diff
@@ -0,0 +1,271 @@
+summary: Create a Keptn Notification Service
```
Comment (johannes-b) [inline site/tutorials/keptn-notification-service_gen.md:1]: Do you want to check in this file?

File: site/tutorials/keptn-argo-cd-deployment/codelab.json (hunk 1 above)
Comment (jetzlstorfer) [reply site/tutorials/keptn-argo-cd-deployment/codelab.json:24]: yes

File: site/tutorials/keptn-notification-service_gen.md (hunk 2 above)
Comment (jetzlstorfer) [reply site/tutorials/keptn-notification-service_gen.md:1]: thanks - the file is a duplication and I will remove it

**Pull request merged**